Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11689,6 +11797,40 @@ end
 					},
 					Notes = "Returns true if the specified player is queued to be transferred to a World.",
 				},
+				RegisterForgeMod =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list.",
```

Comment:
Changed to manual binding for RegisterForgeMod\*, tested with:

```lua
local isSuccess, err = pcall(function () cRoot:Get():GetServer():RegisterForgeMod("foo", "1.2.3") end)
```

in https://github.com/satoshinm/SampleForgeMod.lua/commit/5cd2ea580ad411df96125bd366b4ae476c0c6a05